vendor/vendor.json

```diff
@@ -0,0 +1,2213 @@
+{
```

Comment:
This isn't needed any more (since we use go modules and not dep).

Reply:
Good catch, I'm not sure how this snuck in.